components/dashboard/craco.config.js

```diff
@@ -11,6 +11,21 @@ module.exports = {
             plugins: [require("tailwindcss"), require("autoprefixer")],
         },
     },
+    webpack: {
+        configure: {
+            resolve: { fallback: { crypto: false, net: false, path: false, fs: false, os: false } },
```

Comment:
I think we'll need to pollyfill it here like in supervisor frontend https://github.com/gitpod-io/gitpod/pull/18552